l2geth/core/vm/evm.go

```diff
@@ -425,6 +425,9 @@ func (evm *EVM) create(caller ContractRef, codeAndHash *codeAndHash, gas uint64,
 		return nil, common.Address{}, gas, ErrInsufficientBalance
 	}
 	nonce := evm.StateDB.GetNonce(caller.Address())
+	if nonce+1 < nonce {
+		return nil, common.Address{}, gas, ErrNonceUintOverflow
+	}
 	evm.StateDB.SetNonce(caller.Address(), nonce+1)
```

Comment:
what‘s the purpose of this code?

Reply:
> what‘s the purpose of this code?

https://github.com/mantlenetworkio/mantle/pull/1292

Reply:
Uint64 overflow